### PR TITLE
Added docstrings to Currency Exchange exemplar.py and stub file.

### PR DESCRIPTION
**Pull request by BethanyG**

### CODE REVIEW

**Review by kotp — COMMENTED** [review]
Missing EOL at end of exchange.py

File: exercises/concept/currency-exchange/exchange.py [hunk 1]
```diff
@@ -1,22 +1,54 @@
...
+    '''
+
+    :param budget: float - the amount of your money you are planning to exchange.
+    :param exchange_rate: float - the unit value of the foreign currency.
+    :param spread: int - percentage that is taken as an exchange fee.
+    :param denomination: int - the value of a single bill.
+    :return:
+    '''
+
+    pass
```
Comment (BethanyG) [inline exercises/concept/currency-exchange/exchange.py:54]:
```suggestion
    pass
    
```

File: exercises/concept/currency-exchange/exchange.py [hunk 2]
```diff
@@ -1,22 +1,55 @@
...
+
+    :param budget: float - the amount of your money you are planning to exchange.
+    :param exchange_rate: float - the unit value of the foreign currency.
+    :param spread: int - percentage that is taken as an exchange fee.
+    :param denomination: int - the value of a single bill.
+    :return:
+    '''
+
     pass
+
```
Comment (BethanyG) [inline exercises/concept/currency-exchange/exchange.py:55]:
```suggestion

```

Comment (BethanyG) [inline exercises/concept/currency-exchange/exchange.py]:
```suggestion

    pass
    
```

Comment (BethanyG) [reply exercises/concept/currency-exchange/exchange.py]: Arg.  This is just stuck in a state where I can't add this in!!

**Review by kotp — APPROVED** [review]
Approving so I can have it merged, and will fix the 0x0a character.

**Pull request merged**